nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/resources/OSGI-INF/comment-pageprovider-contrib.xml

```diff
@@ -3,6 +3,14 @@
 
   <extension target="org.nuxeo.ecm.platform.query.api.PageProviderService" point="providers">
 
+    <coreQueryPageProvider name="GET_COMMENT">
```

Comment:
It's more a `GET_COMMENTS_FOR_EXTERNAL_ENTITY`.

Reply:
No, as we want to have a single comment with the external id `entitiId`. Maybe `GET_COMMENT_AS_EXTERNAL_ENTITY` would be better.

Reply:
Indeed 👍